wire-ios-data-model/Source/Model/FeatureConfig/AppLock/AppLockController.swift

```diff
@@ -116,7 +116,11 @@ public final class AppLockController: AppLockType {
 
     // MARK: - Life cycle
 
-    public init(userId: UUID, selfUser: ZMUser, legacyConfig: LegacyConfig? = nil) {
+    public init(
+        userId: UUID,
+        selfUser: ZMUser,
+        legacyConfig: LegacyConfig?
```

Comment:
I like that you removed the default value, I think default values can lead to unsafe code.